### PR TITLE
Add Software Patreons

### PR DESCRIPTION
**Pull request by uraimo**
This PR adds https://github.com/uraimo/awesome-software-patreons , a list of patreon accounts of individual software developers or software projects.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

Yup, read and understood!

### CODE REVIEW

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by aschrijver** [comment]
Hi @uraimo 

I have a couple of suggestions to improve your list (but on the whole your list seems to mostly comply to the current ruleset):

- The Contents section should be the first in the list. You can add Contributing section below that, or at the end of the list
- You have a LICENSE file, but you could add a License section at the end of the list, similar to that in the top-level awesome
- Not sure if this is still a requirement, but the URL in the PR should better end with the `#readme` hash

**Comment by sindresorhus** [comment]
@uraimo ⬆️

**Comment by uraimo** [comment]
All done, now waiting for something new to review :)

**Comment by uraimo** [comment]
I've started reviewing PRs, will adde them below as I progress through the list:
#1358
#1375
#1393
#1319

**Comment by sindresorhus** [comment]
- It's `open source` or `open-source, not `opensource`.
- `iTerm2 - iTerm2.` This is not a good description.
- Descriptions should not be titlecase (unless of course it's a name): `Ubuntu Email Client.` => `Ubuntu email client.`. Applies to many other descriptions too.
- `Bittorrent` => `BitTorrent`
- `ECommerce` => `Ecommerce`
- `Javascript` => `JavaScript`
---

> GIMP - Øyvind Kolås, lead developer of GIMP.

The description should include what GIMP is.

> Improved Initiative - D&D Combat tracker.

Should mention what D&D is.

> Tiled - a 2D game level editor.

Not correctly capitalized.

> Objective-See, various mac security tools and articles.

mac => Mac

> Joey Hess - git-annex, keysafe, other linux software.

linux => Linux

> Ethan Lee - XNA Preservation Tools and Linux/Mac Games

Should end in a dot.

File: readme.md [hunk 1]
```diff
@@ -598,6 +598,7 @@
 - [Jupyter](https://github.com/markusschanta/awesome-jupyter#readme) - Create and share documents that contain code, equations, visualizations and narrative text.
 - [FIRST Robotics Competition](https://github.com/andrewda/awesome-frc#readme) - International high school robotics championship.
 - [Humane Technology](https://github.com/engagingspaces/awesome-humane-tech#readme) - Open source projects that help improve society.
+- [Software Patreons](https://github.com/uraimo/awesome-software-patreons#readme) - Programmers and software-related Patreon accounts.
```
Comment (sindresorhus) [inline readme.md]: This should describe what software patreons are, not what the list contains.

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example)

**Comment by uraimo** [comment]
Thanks for the throughout review, I've fixed similar issues to the one you pointed out elsewhere and improved some project descriptions too.

**Comment by uraimo** [comment]
Fixed conflict, should have been able to do it on its own...

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1049182112031465473

**Pull request merged**